oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/NodeStateEntryTraverser.java

```diff
@@ -127,7 +155,7 @@ private boolean includeDoc(NodeDocument doc) {
 
     private CloseableIterable<NodeDocument> findAllDocuments() {
         return new MongoDocumentTraverser(documentStore)
-                .getAllDocuments(Collection.NODES, id -> includeId(id));
+                .getAllDocuments(Collection.NODES, lastModifiedRange, id -> includeId(id));
```

Comment:
nitpick:  you can use method reference here

```suggestion
                .getAllDocuments(Collection.NODES, lastModifiedRange, this::includeId);
```